### DISCUSSION TITLE
Navigation Column should scroll to and focus on currently viewed page.

### DISCUSSION THREAD

**Discussion by helloimalemur** [GitbookIO/community#642] [Feature Requests]
Enable navigation Column scroll to and focus on currently viewed page.
For example, when you search for a page in Gitbook, the navigation bar currently does nothing. I believe it would be beneficial for the navigation bar to scroll to and select the page that was brought via search.

![Peek 2024-06-13 17-21](https://github.com/GitbookIO/community/assets/37273704/eddca5ff-de16-466a-814c-e874e9a71a0d)

**Comment by electricbolt** [comment]
+1 for this, but it should also work for direct links, for instance, paste the following GitBook hosted page into a new browser: https://docs.electricbolt.co.nz/appfigurate-se-user-guide/introduction
I'd like the left hand menu to auto scroll to the correct location. Thanks.

**Comment by annettemf** [comment]
Yes, this! We have customers writing in assuming that all that exists is the content on the parent page, and I think it's because they are following links we provide to them and then don't scroll in the ToC to find where the page is that they're currently viewing. We have a LOT of sub-pages and visual discoverability is huge for our customer base.

**Comment by aikoii** [comment]
Hey everyone 👋! We've fixed this, and it should now automatically scroll to the currently viewed page when searching or visiting a page directly 🎉

**Comment by helloimalemur** [comment]
Close! It only automatically scrolls if the searched for page has it's drop down path already expanded.
![Peek 2024-10-01 21-10](https://github.com/user-attachments/assets/f1fe29d5-1bbc-4d29-8132-415d01764f16)
If I search for a page that is closed, it does not auto scroll.
![Peek 2024-10-01 21-10-2](https://github.com/user-attachments/assets/b973ae17-a9bc-42e2-ad27-9a9455909c72)

**Comment by denisemcg** [comment]
I agree - it's close but not quite perfect. I still had to scroll down the menu a bit...
https://github.com/user-attachments/assets/e29d9c37-d854-4182-a5ec-df41d4e87ca6
